deploy/01_manager.ts

```diff
@@ -3,7 +3,17 @@ import { DeployFunction } from "hardhat-deploy/types";
 
 const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   const { deploy } = hre.deployments;
-  const { deployer } = await hre.getNamedAccounts();
+
+  // Get a list the named accounts based on network
+  const namedAccounts = await hre.getNamedAccounts();
+
+  const deployer = namedAccounts.deployer;
+
+  console.log("deployer--->", namedAccounts);
```

Comment:
when i run
yarn deploy:alfajores
this does not print anything, but it works for the 00_multisig.ts